package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "broccoli-kitchen-sink-helpers": "^0.2.5",
     "coffee-script": "^1.7.1",
+    "cross-spawn": "^2.0.0",
```

Comment:
Perhaps this should be bumped to a more recent version, cross-spawn has been updated to 6.0.5

Reply:
I think it should be fine, yes. Looks like the api didn't change at all.